src/figuresClasses.ts

```diff
@@ -1,19 +1,89 @@
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
+
+type Color = 'red' | 'green' | 'blue';
+
 export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
+}
 
+function roundNum(num: number): number {
+  return Math.floor(num * 100) / 100;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shape = Shape.Triangle;
 
-}
+  constructor(
+    public color: Color,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Sides of triangle should be greater than zero');
+    }
+
+    if (a >= b + c || b >= c + a || c >= a + b) {
+      throw new Error(
+        'One side of triangle can\'t be greater than sum of other two',
```

Comment:
Come on, just use interpolation
```suggestion
        `One side of triangle can't be greater than sum of other two`,
```

Reply:
If I don't add any variable to interpolation, it appears to be a mistake